testplan/web_ui/testing/src/AssertionPane/AssertionTypes/__tests__/BasicAssertion.test.js

```diff
@@ -22,6 +22,26 @@ function propsEqual() {
   };
 }
 
+function propsHighlightEqual() {
+  return {
+    assertion: {
+      "category": "DEFAULT",
+      "machine_time": "2019-02-12T17:41:42.795536+00:00",
+      "description": "Hel == Hello",
+      "line_no": 25,
+      "label": "==",
+      "second": "Hello",
```

Comment:
Add a more complex example, that utilize more codepaths

first: "This is a short things",
second: "And this is a long things with some extra at the end"